### PR TITLE
Admin manages former collaborators

### LINKED ISSUE #16: Business admin views past project collaborators
As a business admin, 
when I visit "/<business name>", 
and I have added and removed one or more collaborators, 
I view a list with all past collaborators for this business.

### PR DESCRIPTION
**Pull request by Dmitry1007**
close #16

### CODE REVIEW

**Comment by marlabrizel** [comment]
I'm not comfortable merging this right now - there are a lot of changes to how the permissions work.

**Comment by Dmitry1007** [comment]
There's no change to how permissions work at all. I only built one more permission for "former collaborator" and currently his/her permission is exactly the same, and no more, as for a "registered user".

**Comment by marlabrizel** [comment]
we do need tests for the new permissions

**Comment by Dmitry1007** [comment]
word
let me ponder on how to do it without creating another permission.

**Comment by applegrain** [comment]
FINALLY!

File: app/controllers/companies/users_controller.rb [hunk 1]
```diff
@@ -19,11 +18,22 @@ def create
...
 
   def update
-    user = User.find_by(id: params[:id])
-    user.update_attribute("company_id", nil)
-    role = user.roles.where(name: "collaborator")
-    user.roles.delete(role)
-    redirect_to :back
+    if params[:reinstate].nil?
+      user = User.find_by(id: params[:id])
+      # user.update_attribute("company_id", nil)
```
Comment (applegrain) [inline app/controllers/companies/users_controller.rb]: remove comment! :)

**Comment by applegrain** [comment]
Nice job Dmitry. :hamburger:

**Pull request merged**